assets/config_authentication_cloud.json

```
@@ -2,5 +2,17 @@
 	"sdk": {
 		"hostname": "myinstance.mauth.nevis.cloud",
 		"facetId": "android:apk-key-hash:ch.nevis.mobile.authentication.sdk.react.example"
-	}
+	},
+	"authenticatorAllowlist": [
```

🔧 Please check for other places.
```suggestion
	"authenticatorAllowList": [
```